dask/array/core.py

```diff
@@ -842,6 +842,8 @@ def store(sources, targets, lock=True, regions=None, compute=True, **kwargs):
         for the corresponding source and target in sources and targets, respectively.
     compute: boolean, optional
         If true compute immediately, return ``dask.delayed.Delayed`` otherwise
+    keep: boolean, optional
```

Comment:
`keep` doesn't feel like the best keyword, but I struggle to think of a better one. `return_stored`?

Reply:
Yeah I liked `keep` as it was short and seemed clear (but maybe not for everyone). Guess `return` is out. 😜 Could go with `return_stored`.

Reply:
I find `return_stored` clearer, but there may be a better option.

Reply:
Ok, done.